tests/infra/node.py

```diff
@@ -150,6 +150,8 @@ def __init__(
 
         if os.getenv("CONTAINER_NODES"):
             self.remote_shim = infra.remote_shim.DockerShim
+        elif isinstance(host, str) and host.split("://")[0] == "aci":
```

Comment:
It feels like we should decouple the deployment type (ACI) from the protocol used to drive the node (local or SSH). Would it be better to choose to use the `AciShim` via an environment variable, or a new parameter to `Network`?

Reply:
https://github.com/microsoft/CCF/pull/4943/commits/fbe45a81597596398edc34499a3c8867f7e11bf4